### PR TITLE
Templates for subjQA

### PR DESCRIPTION
**Pull request by arnaudstiegler**
All subsets have similar templates

### CODE REVIEW

**Comment by arnaudstiegler** [comment]
Removed the templates using `is_ques_subjective` as it seems ill-defined (not consistent with the definition in HF dataset cards). Opened an issue for that on the datasets GitHub repo

File: templates/subjqa/restaurants/templates.yaml [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    reference: ''
+    task_template: true
+  afe5086e-d9fe-4981-bcac-67d580950110: !Template
+    id: afe5086e-d9fe-4981-bcac-67d580950110
+    jinja: 'Question:
+
+      {{question}}
+
+
+      What is the subjectivity score of the question (on a 1 to 5 scale with 1 being
```
Comment (VictorSanh) [inline templates/subjqa/restaurants/templates.yaml:63]: Reformulation for diversity: `On a scale of 1 to 5 (1 being the most subjective), how subjective is the question?`

**Review by VictorSanh — APPROVED** [review]
minor suggestions, merging once you integrate these

File: templates/subjqa/restaurants/templates.yaml (hunk 1 above)
Comment (arnaudstiegler) [reply templates/subjqa/restaurants/templates.yaml]: Done on all subsets

**Review by VictorSanh — COMMENTED** [review]
thannk you, merging now, please update the spreadsheet

**Pull request merged**